docker/airflow/1.10.3/Dockerfile

```diff
@@ -21,10 +21,9 @@ LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.module="airflow"
 LABEL io.astronomer.docker.component="airflow"
-LABEL io.astronomer.docker.airflow.version="1.10.2"
+LABEL io.astronomer.docker.airflow.version="1.10.3"
 
 ARG ORG="astronomer"
-# ARG VERSION="1.10.2-1"
 ARG VERSION="feature/kube-labels"
```

Comment:
@andriisoldatenko also need to change it here to point at v1-10-3 (the new branch).

Reply:
@schnie fixed.